foodsaving/groups/tests/test_tasks.py

```diff
@@ -0,0 +1,47 @@
+from django.db import DataError
+from django.db import IntegrityError
+from django.test import TestCase
+from django.core import mail
+from django.utils import timezone
+from rest_framework.test import APITestCase
+
+from foodsaving.groups.factories import GroupFactory
+from foodsaving.groups.models import Group, GroupMembership
+from foodsaving.groups.tasks import process_inactive_users
+from foodsaving.users.factories import UserFactory
+from config import settings
+
+from datetime import timedelta
+
+
+# This probably shouldn't be APITestCase???
+class TestProcessInactiveUsers (APITestCase):
+    def setUp(self):        
+        self.admin = UserFactory()  # has membership management rights
```

Comment:
Comment seems wrong. Calling this user `self.active_user` seems more appropriate.

Reply:
Hmmm... I'm not sure that user needs to exist actually. I'm going to remove it.

Reply:
Hang on, it makes sense to keep it. Just to make sure that an active user doesn't get these e-mails :)

Reply:
Have put it back, it now has separate tests for the three different user types